avm/res/compute/virtual-machine-scale-set/README.md

```diff
@@ -362,6 +362,7 @@ module virtualMachineScaleSet 'br/public:avm/res/compute/virtual-machine-scale-s
       enabled: true
     }
     extensionMonitoringAgentConfig: {
+      autoUpgradeMinorVersion: true
```

Comment:
Was the previously failed pipeline expected to release a new version? If so, please also update something minor in main.bicep (e.g., a parameter description) and compile the main.json so that the PR can qualify for publishing

Reply:
@eriqua the version was already updated, but the readme not, so the new version was never published

Reply:
In that case it will continue to not be published, because this PR will not trigger the publishing step.

If we want to fix the missing publishing, we need the main.json to be updated. For that we need a small change in main.bicep and then compile the main.json

Reply:
@eriqua updated the API version, so json is now different. Only one test fails, due to capaciti restrictions